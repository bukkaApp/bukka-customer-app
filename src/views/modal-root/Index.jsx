import React from 'react';

import PropTypes from 'prop-types';

import AuthModal from '../../components/navbar/common/AuthModal';
import InviteFriends from '../invite/InviteFriends';

export const Index = ({ push }) => (
  <>
    <AuthModal push={push} />
    <InviteFriends push={push} />
  </>
);

export default Index;

Index.propTypes = {
  push: PropTypes.func.isRequired,
};
