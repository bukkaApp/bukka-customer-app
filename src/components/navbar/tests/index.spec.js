import React from 'react';

import { Provider } from 'react-redux';
import Index from '..';

describe('Navbar Index container', () => {
  const store = mockStore({
    navbarAuthReducer: { type: 'Sign Up' },
    authenticationReducer: {
      user: {},
      status: {
        authenticated: false,
        error: false
      },
      errorMessage: ''
    },
    deliveryModeReducer: {
      mode: 'pickup'
    }
  });

  const wrapper = shallow(
    <Provider store={store}>
      <Index />
    </Provider>
  );

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
