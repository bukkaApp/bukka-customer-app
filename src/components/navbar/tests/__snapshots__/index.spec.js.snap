// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Index container renders properly 1`] = `ShallowWrapper {}`;
